pkg/controller/appmetricsservice/postgresql.go

```diff
@@ -0,0 +1,199 @@
+package appmetricsservice
+
+import (
+	"fmt"
+
+	metricsv1alpha1 "github.com/aerogear/app-metrics-operator/pkg/apis/metrics/v1alpha1"
+
+	openshiftappsv1 "github.com/openshift/api/apps/v1"
+	"github.com/pkg/errors"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func newPostgresqlPersistentVolumeClaim(cr *metricsv1alpha1.AppMetricsService) (*corev1.PersistentVolumeClaim, error) {
+	pvcSize, err := resource.ParseQuantity("10Gi")
```

Comment:
This should probably be configurable from the CR. Thoughts?

Reply:
Let's just keep it the same with UPS. If SRE people thinks it should be configurable OR if we have time in the future, we can do it.

But it is the ideal situation.